terraform/paas/test.env.tfvars

```diff
@@ -0,0 +1,10 @@
+paas_space                = "get-into-teaching"
+paas_database_common_name = "school-experience-test-pg-common-svc"
+paas_redis_1_name         = "school-experience-test-redis-svc"
+paas_application_name     = "school-experience-app-test"
+paas_internet_hostnames   = ["staging-school-experience"]
```

Comment:
There is a bot of a mix of dev/test/staging. Can we keep just 1? I think this is the dev environment, right?

Reply:
See also test.bk.vars etc

Reply:
this is the test.env so is test, basically the confusion is I am changing test -> staging, so I will change this env file to staging and change the names